index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express')
+
+const app = express()
+const movies = require('./movies.js')
+const parser = require('body-parser')
+app.use(parser.json())
+
+app.get('/movies/:search', (request, response) =>
+{
+  let search = request.params.search.toLowerCase()
+  let filtered = movies.filter((movie) =>
+  {
+    return movie.title.toLowerCase().includes(search) || movie.directors.filter(director => director.toLowerCase().includes(search)).length
+  })
+  response.send(filtered)
+})
+app.get('/movies', (request, response) =>
+{
+  response.send(movies)
+})
+app.post('/movies', (request, response) =>
+{
+  let newMovie = request.body
```

Comment:
This should be `const newMovie` since you don't need to reassign the variable. Always use `const` unless you absolutely have to reassign the variable.